qiita_db/environment_manager.py

```diff
@@ -101,18 +103,21 @@ def _add_ontology_data(conn):
             raise IOError("Error: Could not fetch ontologies file from %s" %
```

Comment:
Not needed. If this is the first function, the transaction didn't even started. If another function inside a transaction is calling this, that other context will catch it and rollback.